docs/CONTRIBUTING.rst

```diff
@@ -9,37 +9,14 @@ All submitted code should follow our `style guidelines
 <https://github.com/secure-systems-lab/code-style-guidelines/blob/master/python.md>`_
 and must be `unit tested <#unit-tests>`_.
 
-Development Installation
-========================
-
-To work on the TUF project, it's best to perform a development install.
-
-To facilitate development and installation of edited version of the code base,
-developers are encouraged to use `venv <https://docs.python.org/3/library/venv.html>`_.
-
-1. First, `install non-Python dependencies
-<https://theupdateframework.readthedocs.io/en/latest/INSTALLATION.html#non-python-dependencies>`_.
-
-2. Then clone this repository:
-
-::
-
-    $ git clone https://github.com/theupdateframework/python-tuf
-
-3. Then perform a full, editable/development install.  This will include all
-   optional cryptographic support, the testing/linting dependencies, etc.
-   With a development installation, modifications to the code in the current
-   directory will affect the installed version of TUF.
-
-::
-
-    $ python3 -m pip install -r requirements-dev.txt
+.. note::
 
+     Also see :ref:`development installation instructions <installation:Install for development>`.
```

Comment:
This reference is not resolved when the rendered document is browsed on GitHub, which is likely to happen because it is the community standard location for a GitHub repo's contributing docs.

A quick fix would be to use the absolute RTD link (see https://github.com/theupdateframework/python-tuf/pull/1781#discussion_r791639408).

In the future we might want to think about separating docs for RTD and docs for GitHub (see #1849)

Reply:
yeah, it does look like sharing docs is tricky and results in issues like this.

Reply:
see cd70e47